frontend/frontend_test.go

```diff
@@ -36,7 +36,7 @@ func TestFrontendIntegration(t *testing.T) {
 	})
 }
 
-func testRefReadFile(t *testing.T, sb integration.Sandbox) {
+func testRefReadFile(t *testing.T, sb integration.Backend) {
```

Comment:
the tests would continue to take `Sandbox`, they need to access all the features and we don't want to rewrite them

Reply:
Fixed.